### PR TITLE
Remove unused dependency and fix changelog

### PR DESCRIPTION
**Pull request by erlendvollset**
- Remove 'coverage' from dependencies
- Remove duplicate section in changelog and fix changelog date

## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change. 
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring. 
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,12 +24,6 @@ Changes are grouped as follows
 ### Added
 - `DetectJobBundle` dataclass: A way to manage multiple files and jobs.
 
-## [4.11.2] - 2022-11-16
-### Changed
-- Detect endpoint (for Engineering Diagram detect jobs) is updated to spawn and handle multiple jobs.
-### Added
-- `DetectJobBundle` dataclass: A way to manage multiple files and jobs.
```
Comment (dmivankov) [inline CHANGELOG.md:31]: was 4.11.12 never released and also its changes not included into 4.11.3?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 - 
-## [4.11.3] - 2022-11-02
+## [4.11.3] - 2022-11-17
```
Comment (dmivankov) [inline CHANGELOG.md:17]: will this pass deployment, or why is the date changed?

File: CHANGELOG.md (hunk 1 above)
Comment (erlendvollset) [reply CHANGELOG.md:31]: It was released. Just removing because it's duplicated.

File: CHANGELOG.md (hunk 2 above)
Comment (erlendvollset) [reply CHANGELOG.md:17]: Date was wrong. This was released today.

**Comment by haakonvt** [comment]
Note: Version `[2.1.1]` also has a duplicate entry in `CHANGELOG.md`:
```
## [2.1.1] - 2020-08-13

### Fixed
- Fixed a bug where only v1 endpoints (not playground) could be added as retryable

## [2.1.1] - 2020-08-04
```

**Review by dmivankov — APPROVED** [review]

**Pull request merged**